ui/index.jsp

```diff
@@ -1761,73 +1761,73 @@
-        <script type="text/javascript" src="scripts/installWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/installWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/projects.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/dashboard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/instanceWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/instanceWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/affinity.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/affinity.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/instances.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/events.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/regions.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/regions.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/ipRules.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/enableStaticNAT.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/securityRules.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/vpc.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/vpc.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/network.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/recurringSnapshots.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/uploadVolume.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/storage.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/templates.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/accountsWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/accountsWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/accounts.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/configuration.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/globalSettings.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/zoneWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/physicalResources.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/zoneWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/system.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/domains.js?t=<%=now%>"></script>
```

Comment:
IE (up to last version) have an option [not to update content](https://support.microsoft.com/en-us/kb/263070) and I believe this is a workaround on that. So at least for IE it is probably still needed.